doc/users/explain/interactive_guide.rst

```diff
@@ -431,10 +431,10 @@ method.  The source for the ``prompt_toolkit`` input hooks lives at
 .. rubric:: Footnotes
 
 .. [#f1] A limitation of this design is that you can only wait for one
-	 input, if there is a need to multiplex between multiple sources
```

Comment:
These don't really need to be changed (and I don't even think we should bother on standardizing indent here).

Reply:
Agreed. We should not have tabs.